editor/run/game_view_plugin.cpp

```diff
@@ -591,6 +595,43 @@ void GameView::_size_mode_button_pressed(int size_mode) {
 	_update_embed_window_size();
 }
 
+void GameView::_reset_time_scales() {
+	time_scale_index = DEFAULT_TIME_SCALE_INDEX;
+	_update_time_scales();
+}
+
+void GameView::_update_time_scales() {
+	Array message;
+	message.append(time_scale_range[time_scale_index]);
+	Array sessions = debugger->get_sessions();
+	for (int i = 0; i < sessions.size(); i++) {
+		if (Object::cast_to<EditorDebuggerSession>(sessions[i])->is_active()) {
+			Ref<EditorDebuggerSession> session = sessions[i];
+			session->send_message("scene:speed_changed", message);
+		}
+	}
+	Color text_color;
+	if (time_scale_index == DEFAULT_TIME_SCALE_INDEX) {
+		text_color = Color(1.0f, 1.0f, 1.0f);
+	} else if (time_scale_index > DEFAULT_TIME_SCALE_INDEX) {
+		text_color = Color(0.0f, 1.0f, 0.0f);
+	} else if (time_scale_index < DEFAULT_TIME_SCALE_INDEX) {
+		text_color = Color(1.0f, 1.0f, 0.0f);
+	}
```

Comment:
Use colors that come from the editor theme, so that they remain readable on a light theme:

```suggestion
	Color text_color;
	if (time_scale_index == DEFAULT_TIME_SCALE_INDEX) {
		text_color = get_theme_color(SNAME("font_color", SNAME("Label"));
	} else if (time_scale_index > DEFAULT_TIME_SCALE_INDEX) {
		text_color = get_theme_color(SNAME("success_color"), EditorStringName(Editor));
	} else if (time_scale_index < DEFAULT_TIME_SCALE_INDEX) {
		text_color = get_theme_color(SNAME("warning_color"), EditorStringName(Editor));
	}
```

Reply:
https://github.com/user-attachments/assets/09fcf4a3-19a5-4069-a0f7-a71ae752c78c

It seems the font color is black

Reply:
Any news? I'm still waiting